src/styles/blog.css

```diff
@@ -72,18 +72,20 @@
 }
 
 .blog-card__image-container {
-  display: flex;
-  justify-content: center;
-  align-items: center;
   overflow: hidden;
   width: 360px;
-  height: 160px;
+  padding-top: 52.5%;
```

Comment:
What is this for?

Reply:
That's the aspect ratio fix, as per [this](https://www.w3schools.com/howto/howto_css_aspect_ratio.asp) article. We want our aspect ratio to be 40:21 ( which is the image's source aspect ratio ), so we set the `padding-top` to `(21/40 * 100)%` which is `52.5%`. It works on chrome, safari and firefox browsers.

Reply:
Can be ignored as we do not need this method anymore